contributed_definitions/nyaml/NXdata_mpes.yaml

```diff
@@ -0,0 +1,245 @@
+category: base
+doc: |
+  :ref:`NXdata_mpes` describes the plottable data and related dimension scales in MPES
+  experiments.
+  
+  It extends the NXdata class and provides a glossary of explicitly named axis names
+  which are typical for MPES data.
+type: group
+NXdata_mpes(NXdata):
+  energy(NX_NUMBER):
+    unit: NX_ENERGY
+    doc: |
+      Calibrated energy axis.
+      
+      This could be a link to either
+      /entry/process/energy_calibration/calibrated_axis or
+      /entry/process/energy_correction/calibrated_axis.
+    \@type:
+      type: NX_CHAR
+      doc: |
+        The energy can be either stored as kinetic or as binding energy.
+      enumeration:
+        kinetic:
+          doc:
+          - |
+            Calibrated kinetic energy axis.
+          - |
+            xref:
+              spec: ISO 18115-1:2023
+              term: 3.35
+              url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:3.35
+        binding:
+          doc:
+          - |
+            Calibrated binding energy axis.
+          - |
+            xref:
+              spec: ISO 18115-1:2023
+              term: 12.16
+              url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.16
+  kx(NX_NUMBER):
+    exists: optional
+    unit: NX_ANY
+    doc: |
+      Calibrated x axis in k-space.
+      Units are 1/Angström.
+  ky(NX_NUMBER):
+    exists: optional
+    unit: NX_ANY
+    doc: |
+      Calibrated y axis in k-space.
+      Units are 1/Angström
+  kz(NX_NUMBER):
+    exists: optional
+    unit: NX_ANY
+    doc: |
+      Calibrated z axis in k-space.
+      Units are 1/Angström.
+  angular0(NX_NUMBER):
+    exists: optional
+    unit: NX_ANGLE
+    doc: |
+      Fast-axis angular coordinate (or second slow axis if angularly integrated).
+  angular1(NX_NUMBER):
+    exists: optional
+    unit: NX_ANGLE
+    doc: |
+      Slow-axis angular coordinate (or second fast axis if simultaneously dispersed in 2 dimensions)
+  spatial0(NX_NUMBER):
+    exists: optional
+    unit: NX_LENGTH
+    doc: |
+      Fast-axis spatial coordinate (or second slow axis if spatially integrated)
+  spatial1(NX_NUMBER):
```

Comment:
We had a long-ish discussion about this in #92. In the end, we decided to go for the option of having `NXdata_mpes` and `NXdata_mpes_detector` base class where we define a glossary of proper MPES-related axis names. However, since base class inheritance is not fully supported yet, we are not using these two base classes in `NXmpes` yet, but rather put the suggested names in the docstrings of the `NXdata` field. This will change in the future, so we have added these two base classes already.